### DISCUSSION TITLE
Allow to auto-change filename on Download

### DISCUSSION THREAD

**Discussion by windigerbus** [shaked6540/YoutubePlaylistDownloader#108] [Ideas]
When you download something it always changes the filename to the video name. It would be nice if you could add prefixes like "Downloaded From YT:" in front of the video title in the filename, and not have to do that in external tools.

**Comment by shaked6540** [comment]
Hey, I added this feature in the new version (1.9.5) please update the program and let me know if this is what you meant. 
Make sure to read the Help section about the File name pattern to understand if thats what you needed

**Reply by windigerbus** [reply]
Yes, this is exactly what I meant. Sorry for the extremely late reply, I didn't see you answered.
